system/update.go

```
@@ -0,0 +1,264 @@
+    sleep 10
+    if systemctl is-active --quiet agent; then
+        rm -f -- "$2"
+        exit 0
+    fi
+fi
+if mv -f -- "$2" "$1"; then
+    chmod 755 "$1"
+    systemctl restart agent
+fi`
+	unit := fmt.Sprintf("reviactyl-agent-update-%d", os.Getpid())
+	command := exec.Command(
```

golangci-lint (2.13.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)